decryption-tool/README.md

```diff
@@ -0,0 +1,15 @@
+# How to run
```

Comment:
Please specify the technical requirements to run the tool.
Which version of Python should be installed. Will it work on Windows, Ubuntu, Mac?